analyzer/src/main/kotlin/Analyzer.kt

```diff
@@ -37,6 +37,7 @@ import org.ossreviewtoolkit.model.config.AnalyzerConfiguration
 import org.ossreviewtoolkit.model.config.RepositoryConfiguration
 import org.ossreviewtoolkit.model.orEmpty
 import org.ossreviewtoolkit.utils.common.CommandLineTool
+import org.ossreviewtoolkit.utils.common.VCS_DIRECTORIES
```

Comment:
The commit message title is a bit unclear, there can only ever be one "single root directory". Maybe say "analyzer: Do not create Unmanaged project if all files are managed".